### DISCUSSION TITLE
Owner root - permissions

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#15004] [How to] [answered]
Somehow I cannot run system if the root is owing files. 
What is recommended access config? What about permissions? 
Appreciate clue-bats

**Comment by rskrzypczak** [comment, marked as answer]
There is nothing strange about this.
The owner doesn't matter. Whatever you chosen or decided upon will have to be configured in the web server configuration files (e.g. Apache2).
However, you should not use the root owner for security reasons.
https://yetiforce.com/en/knowledge-base/documentation/implementer-documentation/item/web-server-requirements
